website/docs/r/container_registry.html.markdown

```diff
@@ -58,6 +58,14 @@ The following arguments are supported:
 
 ~> **NOTE:** The `georeplication_locations` list cannot contain the location where the Container Registry exists.
```

Comment:
COuld we also add the deprecation information to follow the `Optional`? Like this:

* `georeplication_locations` - (Optional / **Deprecated in favor of `georeplications`**) A list of Azure locations where the container registry should be geo-replicated.

Reply:
also can just remove it, but either works